pkg/controllers/provisioning/binpacking/packable.go

```diff
@@ -56,13 +84,11 @@ func PackablesFor(ctx context.Context, instanceTypes []cloudprovider.InstanceTyp
 			packable.validateOperatingSystems(constraints),
 			packable.validateCapacityTypes(constraints),
 			packable.validateAWSPodENI(pods),
-			// Although this will remove instances that have GPUs when
-			// not required, removal of instance types that *lack*
-			// GPUs will be done later.
 			packable.validateNvidiaGpus(pods),
 			packable.validateAMDGpus(pods),
 			packable.validateAWSNeurons(pods),
 		); err != nil {
+			logging.FromContext(ctx).Debugf("Excluding instance type %s because %v", packable.Name(), err.Error())
```

Comment:
The standard pattern in the codebase tends to be:
```suggestion
			logging.FromContext(ctx).Debugf("Excluding instance type %s, %v", packable.Name(), err.Error())
```